apps/settings/src/components/Users/AddExistingUserDialog.vue

```diff
@@ -0,0 +1,125 @@
+<!--
+  - SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+  - SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+<template>
+  <NcDialog class="dialog"
+            size="small"
+            :name="t('settings', 'Add existing account')"
+            out-transition
+            v-on="$listeners">
+    <form id="add-existing-user-form" class="dialog__form" @submit.prevent="submit">
+      <NcSelect v-model="selectedUser"
+                class="dialog__select"
+                :options="possibleUsers"
+                :placeholder="t('settings', 'Search accounts')"
+                :user-select="true"
+                label="displayname"
+                @search="searchUsers" />
+      <NcSelect v-model="selectedGroup"
+                class="dialog__select"
+                :options="availableGroups"
+                :placeholder="t('settings', 'Select group')"
+                label="name"
+                :disabled="preselectedGroup"
+                :clearable="false" />
+    </form>
+    <template #actions>
+      <NcButton class="dialog__submit"
+                form="add-existing-user-form"
+                type="primary"
+                native-type="submit">
+        {{ t('settings', 'Add to group') }}
+      </NcButton>
+    </template>
+  </NcDialog>
+</template>
+
+<script>
```

Comment:
I would prefer new files to be Typescript.
```suggestion
<script lang="ts">
```

(and then use `defineComponent` below)